soh/src/overlays/actors/ovl_En_Go2/z_en_go2.c

```diff
@@ -1341,10 +1344,10 @@ void EnGo2_WakeUp(EnGo2* this, GlobalContext* globalCtx) {
     }
     if ((this->actor.params & 0x1F) == GORON_DMT_BIGGORON) {
         OnePointCutscene_Init(globalCtx, 4200, -99, &this->actor, MAIN_CAM);
-        Animation_ChangeByInfo(&this->skelAnime, sAnimationInfo, ENGO2_ANIM_10);
+        Animation_ChangeByInfo(&this->skelAnime, sAnimationInfo, ((CVar_GetS32("gGoronSpeen", 0) == 1) ? ENGO2_ANIM_10 : ENGO2_ANIM_14));
```

Comment:
Well I intended it as more of an "Easter Egg" that could be enabled since people seemed to like the spinning Gorons. Since it would only be enabled for some silly fun, I was thinking later we find some semi-obscure way to expose it, like a junk hint in Rando or some kind of splash screen message. I could add a checkbox to the cheats menu if you think that would be better?